From f910837dace250418cba4155a6708d47d45075cc Mon Sep 17 00:00:00 2001
From: Nilesh Javili <nilesh.javali@cavium.com>
Date: Thu, 27 Apr 2017 17:59:00 -0700
Subject: iscsid: Add qedi ping transport hook

iscsiuio ping is operational for qedi.
Add missing qedi transport hook for ping support.

Signed-off-by: Nilesh Javali <nilesh.javali@cavium.com>
---
 usr/transport.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/transport.c b/usr/transport.c
index 533ba30a8846..3b7a00a2245e 100644
--- a/usr/transport.c
+++ b/usr/transport.c
@@ -124,6 +124,7 @@ struct iscsi_transport_template qedi = {
 	.ep_poll	= ktransport_ep_poll,
 	.ep_disconnect	= ktransport_ep_disconnect,
 	.set_net_config = uip_broadcast_params,
+	.exec_ping	= uip_broadcast_ping_req,
 };
 
 static struct iscsi_transport_template *iscsi_transport_templates[] = {
-- 
2.9.3

